tests/containers/podman_integration.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman integration
+# Summary: Upstream podman integration tests
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use containers::utils qw(get_podman_version);
+use utils qw(zypper_call);
+use version_utils qw(get_os_release);
+use containers::common;
+
+sub run {
+    select_serial_terminal;
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    install_podman_when_needed($host_distri);
+
+    # Install tests dependencies
+    my @pkgs = qw(apache2-utils bats buildah criu go gpg2 jq make netcat-openbsd openssl podman-remote python3-PyYAML skopeo socat sudo systemd-container);
+    zypper_call "in @pkgs";
+
+    my $podman_bats = get_required_var("PODMAN_BATS");
+    my $opts = "";
+    if ($podman_bats =~ /^rootless/) {
```

Comment:
Is this meant to contain multiple options? If so, should probably check for word boundaries instead. If not, should check for equality

Reply:
> Is this meant to contain multiple options? If so, should probably check for word boundaries instead. If not, should check for equality

It's a quick test to check whether to run as root or not, but I will merge all 4 tests.